### PR TITLE
 akshat5302 patch 1

### PR DESCRIPTION
**Pull request by akshat5302**
Updated the 15.2 and 15.1 states to 'EOL' in the releases.yml file

### CODE REVIEW

**Comment by akshat5302** [comment]
Updated releases.yml file

**Review by hellcp — CHANGES_REQUESTED** [review]
You should create a new entry for the EOL state instead of modifying the existing one

**Review by akshat5302 — COMMENTED** [review]
Added new states as EOL in versions 15.1 and 15.2

**Comment by hellcp** [comment]
Probably worth looking up the actual dates of when both releases were deprecated on [the wiki](https://en.opensuse.org), otherwise there's a dash after releases line in 15.1

**Review by akshat5302 — COMMENTED** [review]
Checked and edited EOL dates according to wiki

**Comment by hellcp** [comment]
releases line still has a dash, please correct it

**Comment by akshat5302** [comment]
> releases line still has a dash, please correct it

I couldn't able to find dash could you just elaborate on the issue

File: _data/releases.yml [hunk 1]
```diff
@@ -28,9 +28,11 @@
     state: 'RC'
   - date: '2020-07-02 11:45:00 UTC'
     state: 'Stable'
+  - date: '2021-12-01 12:00:00 UTC'
+    state: 'EOL'
 - version: 15.1
   order: 4
-  releases:
+  releases:-
```
Comment (hellcp) [inline _data/releases.yml:35]:
```suggestion
  releases:
```

**Review by akshat5302 — COMMENTED** [review]
This change had been done by you I think it's not showing in my file

**Review by hellcp — APPROVED** [review]

**Comment by akshat5302** [comment]
Closing this PR now ??

**Pull request merged**